kvm/requests.yaml

```diff
@@ -0,0 +1,6 @@
+releases:
+    - name: "> 11.3.2, < 12.0.0"
```

Comment:
This one makes sense to me because the reported PM can actually happen as calico does IPAM calculations.